packages/gatsby-plugin-sitemap/src/__tests__/gatsby-node.js

```diff
@@ -12,7 +12,7 @@ beforeEach(() => {
   global.__PATH_PREFIX__ = ``
 })
 
-describe(`Test plugin sitemap`, async () => {
+describe(`Test plugin sitemap`, () => {
   it(`default settings work properly`, async () => {
```

Comment:
do you need to remove async() here too?

Reply:
just for describe blocks, async is still applicable to test blocks (`it`)